version-tracking.edn

```diff
@@ -1,7 +1,17 @@
-[{:created-at "2023-02-09T06:54:29.820472",
+[{:created-at "2023-03-01T15:42:46.624815",
+  :version "23.3.1",
+  :description "Minor fixes to cljs-web3-next",
+  :libs ["shared/cljs-web3-next",
+         "browser/district-ui-web3"],
+  :updated-at "2023-03-01T15:42:46.625332"}
+ {:created-at "2023-02-09T06:54:29.820472",
   :version "23.2.9",
   :description "Import 2 more libraries...",
-  :libs ["shared/district-sendgrid" "browser/district-ui-router" "shared/district-shared-bundle" "browser/district-ui-bundle"],
+  :libs
+  ["shared/district-sendgrid"
```

Comment:
Here you've added 2 new items to the list. The current implementation of `monorepo-tools` will only use the first element to determine what to deploy: https://github.com/district0x/monorepo-tools/blob/master/src/generate_ci_config.clj#L180

The way it is, the `browser/district-ui-router`, `shared/district-shared-bundle` and `browser/district-ui-bundle` wouldn't get released. To fix it, just add them to the `:libs` of the first map.

Reply:
I haven't added those libs. The bb task I used (mark-for-release) apparently reformatted this line, so I just leave it as such. I can revert this line, but then we'll have to manually edit it any time we use the bb tasks.

Reply:
Yeah, that's fine if you didn't mean to release the bundles.